src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineParameters.java

```diff
@@ -0,0 +1,34 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import java.awt.Color;
+
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.line.AbstractLine;
+
+public class LineParameters extends AbstractLine implements ILine{
```

Comment:
Name LineParameters is confusing (maybe ParametrizedLine or LineWithParameters).